Task/python/Taskfile.yaml

```diff
@@ -0,0 +1,134 @@
+# https://taskfile.dev
+
+version: '3'
+
+includes:
+  base: 
+    taskfile: ../Taskfile.yaml
+    internal: true
+
+vars:
+  IMAGE_NAME: null
+  PYTHON_VERSION: null
+  PROJEC_SLUG: null
+  REPO_TAGS:
+    sh: git tag -l
+
+tasks:
+  lint:
+    preconditions:
+      - sh: test `git status -s | wc -w` = "0"
+        msg: "There are untracked files, please commit before linting."
+    cmds:
+      - docker run --rm -v "${PWD}:/goat" -e RUN_LOCAL=true seiso/goat:latest
+
+  test:
+    preconditions:
+      - sh: test `git symbolic-ref -q HEAD`
+        msg: "In detached HEAD state."
+    cmds:
+      - pipenv run pytest tests
+
+  build:
+    vars:
+      VERSION:
+        sh: pipenv run python -c 'from {{.PROJECT_SLUG}} import __version__; print(__version__)'
+      TAG_COMMIT_HASH:
+        sh: git rev-list -1 v{{.VERSION}}
+      COMMIT_HASH:
+        sh: git rev-parse HEAD
+      COMMIT_HASH_SHORT:
+        sh: git rev-parse --short HEAD
+      BUILD_VERSION:
+        sh: |
+          pipenv run python -c '
+          version_string = f"v{{.VERSION}}"
+          repo_tags = []
+          {{range $tag := .REPO_TAGS | splitLines -}}
+          repo_tags.append("{{$tag}}")
+          {{end}}
+          if (version_string in repo_tags and "{{.TAG_COMMIT_HASH}}" == "{{.COMMIT_HASH}}"):
+              build_version = "{{.VERSION}}"
+          else:
+              build_version = f"{{.VERSION}}-{{.COMMIT_HASH_SHORT}}"
+          print(build_version)'
+    cmds:
+      - docker buildx build --tag {{.IMAGE_NAME}}:latest --tag {{.IMAGE_NAME}}:{{.BUILD_VERSION}} .
+
+  reformat:
+    cmds:
+      - >
+        docker run --rm -v "${PWD}:/goat" -e RUN_LOCAL=true
+        --entrypoint isort seiso/goat:latest .
+        --settings-file /action/lib/.automation/.isort.cfg
+      - >
+        docker run --rm -v "${PWD}:/goat" -e RUN_LOCAL=true
+        --entrypoint black seiso/goat:latest .
+
+  update:
+    cmds:
+      - >
+        docker run  --rm -v "${PWD}:/usr/src/app" -w /usr/src/app python:{{.PYTHON_VERSION}} /bin/bash
+        -c "python3 -m pip install --upgrade pipenv &>/dev/null && pipenv update"
+
+  release-calver:
+    preconditions:
+      - sh: test `git status -s | wc -w` = "0"
+        msg: "There are untracked files, please commit before creating a release."
+    vars:
+      DATE_INFO: '{{now | date "2006.01"}}'
```

Comment:
That's an interesting syntax...  Is this YYYY.MM?

Reply:
We should be able to pass a valid `fmt` from [here](https://github.com/go-task/slim-sprig/blob/52ccab3ef572c7e1a2c258be183f9a9296d60152/date.go#L14)